cmake/3rdParty/script-only-mode/PostProcessScriptOnlyMappings.cmake

```
@@ -0,0 +1,112 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+# this file is used during generation of the installer layout.
+# note that ARGV0 is the cmake command itself, and ARGV1 is the -P command line option, so we skip them
+# ARGV2 is the name of this script.
+# ARGV3 is the output folder name
+# ARGV4 and onwards are the names of input files, generated during the engine configure/generate
+# containing information about the 3p libraries available.
+
+#[[ there is 1 input file for each 3p library, and each input file contains a set of "set" commands in this format:
+    set(target_name ${TARGET_NAME})
+    set(copy_dependencies_source_file_path ${copy_dependencies_source_file_path})
+    set(copy_dependencies_target_rel_path ${copy_dependencies_target_rel_path})
+
+    The goal of this file is to generate one output_filename/target_name.cmake for each input file such 
+    that it contains the ability to register the given 3p library target
+    libraries as fake ones (which have the appropriate dependencies so that files get copied).
+]]#
```

This isn't a [cmake bracket](https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#bracket-argument) comment, as they need to start with `[`.

Therefore lines 21-24 will be parse as cmake statements which will cause an error
```suggestion
[[ there is 1 input file for each 3p library, and each input file contains a set of "set" commands in this format:
    set(target_name ${TARGET_NAME})
    set(copy_dependencies_source_file_path ${copy_dependencies_source_file_path})
    set(copy_dependencies_target_rel_path ${copy_dependencies_target_rel_path})

    The goal of this file is to generate one output_filename/target_name.cmake for each input file such 
    that it contains the ability to register the given 3p library target
    libraries as fake ones (which have the appropriate dependencies so that files get copied).
]]
```

---

Are you sure about that?  The manpages you linked to literally say that it starts with a # and then [[
> A # immediately followed by a [bracket_open](https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#grammar-token-bracket_open) forms a bracket comment consisting of the entire bracket enclosure
and their example looks like this
```
#[[This is a bracket comment.
It runs until the close bracket.]]
```
it also shows up as a comment in the cmake syntax highlighter

---

Sorry I was looking at the Bracket Argument syntax and not bracket comment.